app/controllers/signs_controller.rb

```diff
@@ -1,6 +1,6 @@
 class SignsController < ApplicationController
   before_action :set_sign, only: %i[ show edit update destroy ]
-  
+
```

Comment:
@anilpothuri  check these kind of files before pushing the code, it seems this file has no changes, but you've pushed the fle.